dev/com.ibm.ws.security.authorization.jacc/src/io/openliberty/security/authorization/jacc/internal/proxy/JavaSePolicyProxyImpl.java

```diff
@@ -54,6 +58,6 @@ public boolean implies(String contextId, Subject subject, Permission permission)
         } else {
             pd = nullPd;
         }
-        return Policy.getPolicy().implies(pd, permission);
+        return (isJava24OrLater ? policy : Policy.getPolicy()).implies(pd, permission);
```

Comment:
Why not always use the `this.policy` here?  Is there a valid usecase where the `Policy` set in the `Policy` class is different than the one set in `this.policy` AND we want to delegate to it instead?

Reply:
I left it using Policy.getPolicy() since that is what it has done in the past and I didn't want to change the behavior.